### PR TITLE
Run blacken-docs v1.11.0 on the LaTeX pycon blocks

### PR DESCRIPTION
**Pull request by peterjc**
- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit`` locally,
and understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Concludes work started in ##3759, cross reference #3758.

### CODE REVIEW

File: Doc/Tutorial/chapter_learning.tex [hunk 1]
```diff
@@ -78,42 +78,26 @@ \subsection{Training the logistic regression model}
...
+...     [58, -259.87],
+...     [126, -414.53],
+...     [191, -249.57],
+...     [113, -265.28],
+...     [145, -312.99],
+...     [154, -213.83],
+...     [147, -380.85],
+...     [93, -291.13],
+... ]
+>>> ys = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0]
```
Comment (peterjc) [inline Doc/Tutorial/chapter_learning.tex:100]: We can keep this as one value per line if preferred by adding a trailing comma (black convention).

File: Doc/Tutorial/chapter_align.tex [hunk 2]
```diff
@@ -3410,15 +3411,15 @@ \subsection*{Reading \texttt{Array} object from file}
...
-H -2  0  1 -1 -3  0  0 -2  8 -3 -3 -1 -2 -1 -2 -1 -2 -2  2 -3  0  0 -1 -4 
+A  4 -1 -2 -2  0 -1 -1  0 -2 -1 -1 -1 -1 -2 -1  1  0 -3 -2  0 -2 -1  0 -4
+R -1  5  0 -2 -3  1  0 -2  0 -3 -2  2 -1 -3 -2 -1 -1 -3 -2 -3 -1  0 -1 -4
+N -2  0  6  1 -3  0  0  0  1 -3 -3  0 -2 -3 -2  1  0 -4 -2 -3  3  0 -1 -4
+D -2 -2  1  6 -3  0  2 -1 -1 -3 -4 -1 -3 -3 -1  0 -1 -4 -3 -3  4  1 -1 -4
+C  0 -3 -3 -3  9 -3 -4 -3 -3 -1 -1 -3 -1 -2 -3 -1 -1 -2 -2 -1 -3 -3 -2 -4
+Q -1  1  0  0 -3  5  2 -2  0 -3 -2  1  0 -3 -1  0 -1 -2 -1 -2  0  3 -1 -4
+E -1  0  0  2 -4  2  5 -2  0 -3 -3  1 -2 -3 -1  0 -1 -3 -2 -2  1  4 -1 -4
+G  0 -2  0 -1 -3 -2 -2  6 -2 -4 -4 -2 -3 -3 -2  0 -2 -2 -3 -3 -1 -2 -1 -4
+H -2  0  1 -1 -3  0  0 -2  8 -3 -3 -1 -2 -1 -2 -1 -2 -2  2 -3  0  0 -1 -4
```
Comment (peterjc) [inline Doc/Tutorial/chapter_align.tex:3438]: My editor dropped the trailing spaces - seems harmless here?

**Comment by peterjc** [comment]
Will rebase - looks like some doctest white spaces issues, probably the editor Atom being too "helpful".

**Comment by peterjc** [comment]
It may be worth explicitly setting blacken-docs to use ``-l`` or ``--line-length`` to a lower value that the black default, since for documentation we often want shorter lines for display on screen or PDF?

**Review by mdehoon — APPROVED** [review]

**Pull request merged**